Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -97,9 +97,7 @@ public class CustomerIOImplementation: CustomerIOInstance {
 
         let queueStatus = backgroundQueue.addTask(type: QueueTaskType.identifyProfile.rawValue,
                                                   data: queueTaskData,
-                                                  groupsParent: [QueueTaskGroup
-                                                      .identifiedProfile(identifier: identifier).string],
-                                                  groupsChild: nil)
+                                                  groupStart: .identifiedProfile(identifier: identifier))
```

Comment:
`groupStart` doesn't feel quite right either, since it's more like a critical task in the group, doesn't necessarily need to be the start.. but unless another idea comes to mind I'm fine with it